workbench-core/environments/package.json

```diff
@@ -41,18 +41,18 @@
   },
   "dependencies": {
     "@aws-cdk/aws-iam": "^1.159.0",
-    "@aws-sdk/client-cloudformation": "^3.118.1",
-    "@aws-sdk/client-cognito-identity-provider": "^3.118.1",
-    "@aws-sdk/client-dynamodb": "^3.118.1",
-    "@aws-sdk/client-ec2": "^3.119.0",
-    "@aws-sdk/client-eventbridge": "^3.118.1",
-    "@aws-sdk/client-iam": "^3.118.1",
-    "@aws-sdk/client-lambda": "^3.82.0",
-    "@aws-sdk/client-s3": "^3.118.1",
-    "@aws-sdk/client-service-catalog": "^3.118.1",
-    "@aws-sdk/client-ssm": "^3.118.1",
-    "@aws-sdk/client-sts": "^3.118.1",
-    "@aws-sdk/util-dynamodb": "^3.87.0",
+    "@aws-sdk/client-cloudformation": "^3.212.0",
```

Comment:
Out of curiosity, this was the TS typing fix? Was there anything else that we needed to do besides updating to `3.212.0`?